src/theory/sets/theory_sets_private.cpp

```diff
@@ -850,7 +850,7 @@ void TheorySetsPrivate::addCarePairs(TNodeTrie* t1,
     {
       Node f1 = t1->getData();
       Node f2 = t2->getData();
-      if (!d_state.areEqual(f1, f2))
+      if (f1.getKind() == MEMBER || !d_state.areEqual(f1, f2))
```

Comment:
Can you add some comments about what we discussed?  That member should not be treated like a function symbol when building the care graph. Maybe give a very basic example we discussed.

Reply:
Done.